Custom command aliasing

za419 opened the pull request:
This pull request will add the ability to configure an alias for custom commands within their own category.

Demostration:
![image](https://user-images.githubusercontent.com/15851480/104549537-b6ad6780-55f8-11eb-82a7-3cdd3f537676.png)

This PR fixes #79

za419 committed:
Refactor Cadence Cadence to alias Cadence cadence

za419 committed:
Implement aliasing of equalTo custom commands

za419 committed:
Allow disabling equalTo aliases

za419 committed:
Implement remaining aliasing for custom commands

Copy-paste from equalTo

JunhaoSLi commented on bot.js:
```diff
@@ -1406,29 +1426,55 @@ function command(message) {
                     // Either random or format must be present. Prefer random if both exist.
-                    if (operation.random) {
-                        output = selectOne(operation.random);
+                    if (customCommand.random) {
```
do you need another `customCommand.disabled` check around here for when the non-alias command is used directly? It looks like you moved the disabled check for equalTo commands but not targeted, multiTargeted, and startsWith.

za419 replied:
Hm. It seems you are correct. Good catch.

za419 committed:
Fix disabling most customCommands directly

Jun noticed a logic error I made where I checked whether an alias or an
alias target was disabled, but not whether a directly referenced command
was disabled, for startsWith, targeted, and multitargeted commands.

I probably made this error reasoning that I had already checked disabled
once above, but did not consider that this branch had two requirements -
And thus not having an alias would be the same as not being disabled.
